interview-questions.md

```diff
@@ -8,50 +8,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is a variable that can be inside a function that lets us apply in information inside to provide an outpout. The information that gets brought in is the argument that the parameter represents.
 
-Researched answer:
+Researched answer: Parameters are the names of variable(s) that are within a functiion. Arguments are the value(s) that are passed in throught the use of the parameter to execute an output.
 
```

Comment:
Parameters are place holders while arguments are values. Parameters allow you to dynamically accept any values as arguments within the function call aka invoking the function.